services/openshift/scripts/openshift_provision

```diff
@@ -161,25 +163,17 @@ if [ ! -f ${ORIGIN_DIR}/configured.templates ]; then
   # - openshift/nodejs-ex
   # - jboss-openshift/application-templates
   ose_tag=ose-v1.2.0
-  template_list=(
-    # ImageStreams
-    /opt/adb/openshift/templates/image-streams-rhel7.json
-    /opt/adb/openshift/templates/jboss-image-streams.json
-    /opt/adb/openshift/templates/wildfly-image-streams.json
-
-    # EAP
-    /opt/adb/openshift/templates/eap64-mysql-persistent-s2i.json
-    /opt/adb/openshift/templates/eap64-basic-s2i.json
-
-    # WebApp
-    /opt/adb/openshift/templates/jws30-tomcat7-mysql-persistent-s2i.json
-    /opt/adb/openshift/templates/cakephp.json
-    /opt/adb/openshift/templates/cakephp-mysql.json
-
-    # Node.js
-    /opt/adb/openshift/templates/nodejs-mongodb.json
-    /opt/adb/openshift/templates/nodejs.json
-  )
+  template_list=$(find /opt/adb/openshift/templates/common -type f)
+  template_list_adb=$(find /opt/adb/openshift/templates/adb -type f)
+  template_list_cdk=$(find /opt/adb/openshift/templates/cdk -type f)
+  case "${VARIANT_ID}" in
+    "adb")
+      template_list+=" ${template_list_adb}";;
+    "cdk")
+      template_list+=" ${template_list_cdk}";;
+    *)
+      (>&2 echo "[WARN] Unknown VARIANT_ID=\"${VARIANT_ID}\"!");;
+  esac
```

Comment:
Well, @LalatenduMohanty pointed me to this file (and line) on IRC and wanted this done thought /etc/os-release. I am ok with both options.

> https://gist.github.com/praveenkumar/5a8de7499f6a11e8eaaf1019581b79e5

The only think I would change is copying it to the same directory. I there will be 2 files with the same name in common/ and adb/ directories this will most probably overwrite the fist one without a warning or fail.